db/migrate/20220114231625_create_likes.rb

```diff
@@ -0,0 +1,5 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
```

Comment:
- [ ] We are missing the references to user table and to post table from likes, we need to know which post is this like referring to, and who liked that given post, that's why here you need a foreign key to table users and the other one to table posts